### PR TITLE
dev-atsg-20-explicit-height-width &&& dev-atsg-60-Pay-per-case-call-now

### PR DESCRIPTION
**Pull request by Renesauve**
explicit width + height not on image error

#20 

I fixed this on homepage logo although there's more on other pages, still going after them

### CODE REVIEW

**Comment by Renesauve** [comment]
Hey! this should be all good now, I went through every page so hopefully I got them all :D

File: site/src/components/header/SiteBranding.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+export const SiteBranding = ({ title, ...rest }) => {
+  const { logo } = useStaticQuery(
+    graphql`
+      {
+        logo: file(relativePath: { eq: "logo.png" }) {
+          relativePath
+          childImageSharp {
+            gatsbyImageData(
+              layout: FIXED
+              placeholder: BLURRED
```
Comment (carlosloureda) [inline site/src/components/header/SiteBranding.js:15]: The blurred effect hurts visually the visitor's eye. @Renesauve which are the possibilities to show the logo faster but with better quality?

Also check mobile, it is too big:



bad-logo-mobile


Nice improvement on **accesibility**!
Screenshot from 2021-05-31 20-48-10

**Review by carlosloureda — CHANGES_REQUESTED** [review]
Everything looks awesome but we need to improve the logo image on the header on mobile and also can we improve the first rendering of it?

Thanka! Great work :muscle:

File: site/src/components/header/SiteBranding.js (hunk 1 above)
Comment (Renesauve) [reply site/src/components/header/SiteBranding.js:15]: Ah yes! I will fix this, I overlooked mobile, good thing you checked!

Comment (carlosloureda) [reply site/src/components/header/SiteBranding.js:15]: It was while running the ligthhouse that i saw it :D. Also check the blurry effect on load (if we can make something better, Roberto won't like that as it is strange). Let me know your thoughts

**Comment by Renesauve** [comment]
this should be fixed now,

All of our options for the image loading are "TRACED_SVG, BLURRED, DOMINANT_COLOR, or NONE"

I think the TRACED_SVG looks the best but it's up to Robert

**Review by carlosloureda — APPROVED** [review]
Great job. Taking it to dev

**Pull request merged**